modules/tensor_mechanics/src/materials/LAROMANCEStressUpdateBase.C

```diff
@@ -156,8 +156,17 @@ LAROMANCEStressUpdateBaseTempl<is_ad>::LAROMANCEStressUpdateBaseTempl(
     _extrapolation(this->template declareProperty<Real>("ROM_extrapolation")),
 
     _derivative(0.0),
-    _old_input_values(3)
+    _old_input_values(3),
+    _wall_dislocations_step(this->template declareGenericProperty<Real, is_ad>(
+        this->_base_name + "wall_dislocations_step")),
+    _cell_dislocations_step(this->template declareGenericProperty<Real, is_ad>(
+        this->_base_name + "cell_dislocations_step")),
+    _plastic_strain_increment(RankTwoTensor::initNone),
```

Comment:
Why `initNone` if you zero it out below? You could just do 
```suggestion
    _plastic_strain_increment(),
```

Reply:
Ok